src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="page">
+    <div className="page-content">
+      <div className="movies">
+
+        {movies.map(item => (
+          <MovieCard {...item} key={item.imdbId} />
+        ))}
+
+      </div>
+    </div>
+
+    <div className="sidebar">
+      Sidebar will be here
+    </div>
+  </div>
```

Comment:
return everything except movies.map back to App component, MovieList should render a list of movies

Reply:
Fixed!